### PR TITLE
use a bucket policy to grant permissions to access the bucket

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1734.surge.sh

File: modules/ROOT/partials/aws-s3-credentials.adoc [hunk 1]
```diff
@@ -21,9 +20,12 @@ The output should look like this:
 [result,shell]
 ----
 [default]
-region=eu-north-1
+region=us-east-1
 ----
-.. Verify that the `~/.aws/credentials` is correct:
+. Configure the access to your AWS S3 bucket by either setting the `aws_access_key_id` and `aws_secret_access_key` in the `~/.aws/credentials` file or by using a bucket policy.
```
Comment (docrozza) [inline modules/ROOT/partials/aws-s3-credentials.adoc:25]: You'll always need something like the `aws_access_key_id` and `aws_secret_access_key` for authentication

The bucket policy is more for authorisation once your auth'd in the system

File: modules/ROOT/partials/aws-s3-credentials.adoc [hunk 2]
```diff
@@ -36,4 +38,32 @@ The output should look like this:
 [default]
 aws_access_key_id=this.is.secret
 aws_secret_access_key=this.is.super.secret
+----
+
+* Alternatively, you can use a resource-based policy to grant access permissions to your S3 bucket and the objects in it.
```
Comment (docrozza) [inline modules/ROOT/partials/aws-s3-credentials.adoc:43]: This is more 'additionally' rather than alternatively

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1734.surge.sh

**Comment by renetapopova** [comment]
@docrozza, could you be able to take a second look here? Thanks!

**Review by docrozza — APPROVED** [review]

**Pull request merged**